registration-processor/registration-processor-notification-service-impl/src/test/java/io/mosip/registration/processor/notification/service/test/NotificationServiceTestApplication.java

```diff
@@ -0,0 +1,28 @@
+package io.mosip.registration.processor.notification.service.test;
+
+import javax.persistence.EntityManagerFactory;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import io.mosip.kernel.dataaccess.hibernate.config.HibernateDaoConfig;
+
+/**
+ * Audit manager application
+ * 
+ * @author Dharmesh Khandelwal
+ * @since 1.0.0
+ *
+ */
+@SpringBootApplication(scanBasePackages =  { "io.mosip.registration.processor.notification.service.*"},exclude = {HibernateDaoConfig.class} )
+public class NotificationServiceTestApplication {
```

Comment:
What does this class do ?

Reply:
This is a test app class this class just create context when running test junit